pallets/lbp/src/lib.rs

```diff
@@ -768,8 +764,8 @@ impl<T: Config> Pallet<T> {
 				&& pool_data.initial_weight < MAX_WEIGHT
 				&& !pool_data.final_weight.is_zero()
 				&& pool_data.final_weight < MAX_WEIGHT,
-				// TODO people could leak value out the pool if initial weight is < final weight due to fee structure
-				// && pool_data.initial_weight > pool_data.final_weight,
+			// TODO people could leak value out the pool if initial weight is < final weight due to fee structure
```

Comment:
Is all of this rustfmt ? If yes can we do separate PR or will it merge conflict?

Reply:
If we don't have comprehensive rustfmt settings that check this on PRs we should add this (create new issue) to prevent merge conflicts in the future.

Reply:
Yes, I used `make format` cmd. I'll be merging master into this branch so I can undo this all the format changes if we want